tests/e2e/specs/BoundaryTimerEvent.spec.js

```diff
@@ -59,20 +61,29 @@ describe('Boundary Timer Event', () => {
       });
   });
 
-  it('Can drag boundary timer events over valid targets', function() {
+  it('Can add boundary timer events to valid targets', function() {
     const initialNumberOfElements = 1;
     const startEventPosition = { x: 150, y: 150 };
 
-    dragFromSourceToDest(nodeTypes.boundaryTimerEvent, startEventPosition);
+    getElementAtPosition(startEventPosition).then($startEvent => {
+      cy.wrap($startEvent).find('[data-test="boundary-event-dropdown"]').should('not.exist');
+    });
+
     getGraphElements().should('have.length', initialNumberOfElements);
 
     const validBoundaryTimerEventTargets = [
       { type: nodeTypes.task, position: { x: 100, y: 300 } },
       { type: nodeTypes.subProcess, position: { x: 240, y: 300 } },
-      { type: nodeTypes.task, position: { x: 380, y: 300 }, selector: '[data-test=switch-to-script-task]' },
-      { type: nodeTypes.task, position: { x: 100, y: 400 }, selector: '[data-test=switch-to-manual-task]' },
-      { type: nodeTypes.sendTweet, position: { x: 240, y: 400 } },
-      { type: nodeTypes.taskWithMarker, position: { x: 380, y: 400 } },
```

Comment:
Removed `Send Tweet` and `Task with Marker`. They do not have the boundary event dropdown.